l3embedding/training_utils.py

```diff
@@ -0,0 +1,170 @@
+# Copy of https://github.com/fchollet/keras/blob/master/keras/utils/training_utils.py
+# Move import tensorflow as tf to the top to address the pickle issue
```

Comment:
What is the pickle issue?

Reply:
with that import line within the multi-gpu function, the whole model becomes unserializable.